### ISSUE TITLE
list the `[done/total]` before the package name while `bun install`

### ISSUE THREAD

**Issue by pvinis** [oven-sh/bun#480]
it will make it much easier to read while its installing stuff.

ideally, the done part should have enough padding so that it doesn't actually move while counting up.

**Comment by pvinis** [comment]
i would actually love to try this one out, but I've spent like 2 hours trying to find where i need to change this and still don't know haha. any pointers would be appreciated!

so far i have seen that the `install.zig` file is there this happens, i know `setCompletedItems` is used to update, and i saw that `Output.pretty..` is used to print. but I'm still not sure how/where the actual `done/total` is printed 🤔

**Comment by kabaloking** [comment]
The file you need might be 'bun/src/cli/install.sh' 
[This file](https://github.com/oven-sh/bun/blob/418972b42360e6c48f104aff1deaa75b54d7de39/src/cli/install.sh)

**Comment by Omer-Shahar** [comment]
Adding a screenshot:
![image](https://user-images.githubusercontent.com/38870042/178575104-56008b74-5cbc-4917-bfa7-07e5f7da8d75.png)
At least bun is so fast, so you can't really read it in real-time anyway… 😉

**Comment by jdmarshall** [comment]
So are we looking for:

```
[107/203] bun install v0.1.5
  or
bun install [107/203] v0.1.5
  or 
bun install v0.1.5 [107/203]
```
The description suggests the first option, but my instinct says the final one is the easiest to read in a scan of the CLI. 
In particular because any field that can change size makes it difficult to latch on to any other fields around it that might move.  In the final position on the line, it doesn't matter if it resizes over time. The rest of the data is visually stable.

**Comment by jdmarshall** [comment]
I agree with @pvinis - you have a formatted output style and searching the code does reveal where the formatting takes place.  

Looks like it's depending on a standard library for Zig: https://ziglang.org/documentation/master/std/#root;Progress.Node

Which appears to support progress on children, not on the parent? I'll have to look a bit harder, but I think this story is on the cusp of too hard for a first PR, if not past it.
